[Bug] Linux build for 21.5.17 Omega fails with kodi 21.3

### Describe the problem

It does work with the previous addon version with kodi 21.3.

### Possible fix

_No response_

### Steps to reproduce

1. build the addon with kodi v21.3 sources

Packagebuild:

```
# Maintainer: acidicX
pkgname=kodi-addon-inputstream-adaptive-any
pkgver=21.5.17
pkgrel=1
pkgdesc="Inputstream adaptive without kodi-dev dependency, so it can be built on any architecture"
_koditarget=Omega
_gitname=inputstream.adaptive
_kodiver=21.3
arch=('any')
url="https://github.com/xbmc/${_gitname}"
license=('GPL')
groups=('kodi')
makedepends=('cmake' 'git' 'patch')
provides=('kodi-addon-inputstream-adaptive')
conflicts=('kodi-addon-inputstream-adaptive' 'kodi-addon-inputstream-adaptive-19');
depends=('kodi')
source=("https://github.com/xbmc/${_gitname}/archive/${pkgver}-${_koditarget}.tar.gz"
        "https://github.com/xbmc/xbmc/archive/${_kodiver}-${_koditarget}.tar.gz"
)
sha256sums=('f07c72cc0aa4b379a41c991adae7ccbb31ba5f94da13e986d84e53b67e332bdc'
            'a60afcb556e4a00cb19fc35b7e77758107c9a8096c6ce9b66af5f92396be31aa')

prepare() {
        cd xbmc-${_kodiver}-${_koditarget}
}

build() {
        mkdir -p "${_gitname}-${pkgver}-${_koditarget}/build"
        cd "${_gitname}-${pkgver}-${_koditarget}/build"

        cmake \
                -DCMAKE_INSTALL_PREFIX=/usr \
                -DCMAKE_INSTALL_LIBDIR=/usr/lib/kodi \
                -DCMAKE_BUILD_TYPE=Release \
                -DBUILD_SHARED_LIBS=1 \
                -DADDONS_TO_BUILD=${_gitname} \
                -DADDONS_SRC_PREFIX=../.. \
                ../../xbmc-${_kodiver}-${_koditarget}/cmake/addons
        make
}

package() {
        cd "${_gitname}-${pkgver}-${_koditarget}/build"
        install -d "${pkgdir}/usr"
        mv .install/lib "${pkgdir}/usr/"
        mv .install/share "${pkgdir}/usr/"
}
```

### Debug log

```
[ 80%] Performing forcebuild step for 'inputstream.adaptive'
Force build of inputstream.adaptive
[ 82%] Performing build step for 'inputstream.adaptive'
[  3%] Built target cdm_library
[ 11%] Built target mpegts
[ 26%] Built target webm_parser
[ 27%] Building CXX object CMakeFiles/inputstream.adaptive.dir/src/common/AdaptiveStream.cpp.o
In file included from /buildpath/src/inputstream.adaptive-21.5.17-Omega/build/build/inputstream.adaptive/src/common/AdaptiveStream.h:11,
                 from /buildpath/src/inputstream.adaptive-21.5.17-Omega/build/build/inputstream.adaptive/src/common/AdaptiveStream.cpp:9:
/buildpath/src/inputstream.adaptive-21.5.17-Omega/build/build/inputstream.adaptive/src/common/AdaptiveUtils.h:31:11: error: 'uint16_t' does not name a type
   31 | constexpr uint16_t PSSHSET_POS_DEFAULT = 0;
      |           ^~~~~~~~

```

and a lot more `uint16_t` errors

### Stream manifest file(s)

_No response_

### Additional info

_No response_

### Operating system(s)

Linux

### Operating system version(s)

Arch x64 and arm64

### InputStream Adaptive version(s)

21.5.17

### Kodi version(s)

21.3

--- a/src/common/AdaptiveUtils.h
+++ b/src/common/AdaptiveUtils.h
@@ -13,6 +13,7 @@
 #include <memory>
 #include <string_view>
 #include <vector>
+#include <cstdint>
 
 // forwards
 class AP4_Movie;

